--- DirectX-Headers-1.602.0/meson.build~	2022-04-29 20:51:11.000000000 +0000
+++ DirectX-Headers-1.602.0/meson.build	2022-07-06 16:40:55.310483057 +0000
@@ -22,10 +22,6 @@
     meson.override_dependency('DirectX-Headers', dep_dxheaders)
 endif
 
-if not meson.is_subproject() and get_option('build-test')
-    subdir('test')
-endif
-
 pkg = import('pkgconfig')
 pkg.generate(name : 'DirectX-Headers',
              description : 'Headers for using D3D12',
